podcasts/PlaybackManager.swift

```diff
@@ -81,6 +81,7 @@ class PlaybackManager: ServerPlaybackDelegate {
         NotificationCenter.default.addObserver(self, selector: #selector(updateExtraActions), name: Constants.Notifications.extraMediaSessionActionsChanged, object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(updateNowPlayingInfo), name: Constants.Notifications.userEpisodeUpdated, object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(updateAllNowPlayingData), name: .episodeEmbeddedArtworkLoaded, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(handleEpisodeStarredChanged), name: Constants.Notifications.episodeStarredChanged, object: nil)
```

Comment:
Instead of adding a notification, we can look at using 
[`PlaybackManager.nowPlayingStarredChanged`](https://github.com/Automattic/pocket-casts-ios/blob/5866f0f53731c790f9c60d1717e26929ff8aea27/podcasts/PlaybackManager.swift#L1797) which should be triggered in the same way.

Reply:
Good catch! Updated in https://github.com/Automattic/pocket-casts-ios/pull/1195/commits/752451be3d68167b8fdca8f07c77d9c75fd7c3ea